chain-abci/src/staking/tx.rs

```diff
@@ -35,14 +35,19 @@ impl StakingTable {
             return Err(NodeJoinError::BondedNotEnough.into());
         }
 
-        let keypackage = KeyPackage::read_bytes(&tx.node_meta.confidential_init.keypackage)
-            .ok_or(NodeJoinError::KeyPackageDecodeError)?;
-        let info = keypackage
-            .verify(&ENCLAVE_CERT_VERIFIER, block_time)
-            .map_err(NodeJoinError::KeyPackageVerifyError)?;
-        let new_isv_svn = if info.quote.report_body.isv_svn > recent_isv_svn {
-            warn!("more recent version of enclave");
-            info.quote.report_body.isv_svn
+        // FIXME
+        let new_isv_svn = if &tx.node_meta.confidential_init.keypackage != b"FIXME NODE JOIN" {
```

Comment:
I'm not sure whether this can be "fixed" for the unit tests, as they may be executing on non-sgx where one can't obtain a keypackage with valid attestation payload... instead of hardcoding this on the common path, perhaps this could be put in/offloaded to the enclave bridge?
In the real environment, it'll run the full verification; in the mock, it'll skip it or run some reduced version?

Reply:
Currently, the unit test can provide pre-created keypackages, with adjusted reference time(genesis/block time), it runs ok.
But for integration tests, the reference time can't adjusted easily(generated by tendermint process), so it'll be either real one or other mock solution.
The `b"FIXME NODE JOIN"` is only here for integration tests temporarily.